icu4c/source/i18n/indiancal.cpp

```diff
@@ -141,52 +140,18 @@ int32_t IndianCalendar::handleGetYearLength(int32_t eyear) const {
  * @param date The date in Gregorian day in month
  */
 static double gregorianToJD(int32_t year, int32_t month, int32_t date) {
-   double julianDay = (JULIAN_EPOCH - 1) +
-      (365 * (year - 1)) +
-      uprv_floor((year - 1) / 4) +
-      (-uprv_floor((year - 1) / 100)) +
-      uprv_floor((year - 1) / 400) +
-      uprv_floor((((367 * month) - 362) / 12) +
-            ((month <= 2) ? 0 :
-             (isGregorianLeap(year) ? -1 : -2)
-            ) +
-            date);
-
-   return julianDay;
+   return Grego::fieldsToDay(year, month - 1, date) + kEpochStartAsJulianDay - 0.5;
```

Comment:
Isn't month already 0-based here? Should the parameter just be month instead of month-1?

Reply:
(Not completely sure...)

Reply:
I switch to 0 based month in that file now.